### DISCUSSION TITLE
The line number margin changes after reset (very small issue)

### DISCUSSION THREAD

**Discussion by gabdub** [orbitalquark/textadept#183] [General]
Hi,
I noticed that after calling reset() the line number margin shrinks a bit.
I solved the issue by adding:

`events.connect(events.RESET_AFTER, resize_line_number_margin)`

in the main init.lua file (line 170).

I hope it can be added to TA.

**Comment by orbitalquark** [comment]
Thanks for the report. Fixed via https://github.com/orbitalquark/textadept/commit/84c7e9d01390100ebba39d353049da5fbbe32e79
